### DISCUSSION TITLE
隔壁AI討論課程有同學去參加黑克松，有趣的比賽經驗分享

### DISCUSSION THREAD

**Discussion by chia-kai-chang** [1101-GS4719-C/Python-Community-GS4719-C#832] [General]
隔壁AI討論課程有同學去參加黑克松，有趣的比賽經驗分享

網址如下：
https://github.com/NCU-GS3073-AI-Introduction/ai-introduction-community/discussions/482

真是有趣的經驗，也希望之後大家有機會可以多參加類似的比賽活動

**Comment by StevenWanglolz** [comment]
看起來好好玩! 好想去體驗看看，而且現場還有狗勾!!!! 不過寫不寫得出來就是其次了哈哈哈

**Comment by Evan0979** [comment]
之前一直聽黑克松黑克松,但不知道那是什麼,現在終於知道了哈哈

**Comment by peggy0329** [comment]
第一次看到原來黑客松比賽現場是這樣！
還有提供許多娛樂跟用餐服務，真的蠻不錯的
另外，有看到財管的同學也有參加比賽
下次也想參加看看黑客松

**Comment by dinosaur1009** [comment]
有機會的話也想要去試試！但感覺要先精進自己的能力

**Comment by ru0372572** [comment]
好像聽說有同學去黑客松都在當食物小偷，來吃東西順便比賽

**Comment by harrylee1971** [comment]
小時候有去參加過sitcon的黑克松，那時候還小所以不太清楚，現在到大學了若有機會還想再去玩一次看看

**Comment by HsuChen88** [comment]
感覺能在一次比賽中學到很多欸，很令人嚮往。但對於現在的自己來說似乎有點遙遠，更何況沒有能夠一起奮鬥的組員，希望有天也能提槍上陣。
